### DISCUSSION TITLE
Compartilhando conteúdos no Github

### DISCUSSION THREAD

**Discussion by YasminCury** [community/community#137575] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Como posso compartilhar conteúdos no Github?

**Comment by AIExxplorer** [comment]
Você pode estar participando da comunidade aqui mesmo, ou criando repositórios open-source com conteúdos que já desenvolveu ou está desenvolvendo, assim você permite que outras pessoas inclusive possam colaborar no seu projeto!

**Reply by LiteBrite82** [reply]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
